roles/pulp_common/tasks/install_pip.yml

```diff
@@ -128,6 +128,8 @@
         src: templates/requirements.in.j2
         dest: "{{ pulp_install_dir }}/requirements.in"
         mode: '0644'
+      tags:
+        - molecule-idempotence-notest
```

Comment:
Do we want to test this or not?

I thought it was part of being idempotent. We do not want to accidentally upgrade people's installs.

Reply:
Galaxy is a different plugin, in a 1st run it would add just galaxy_ng
but on the 2nd run it would add galaxy_ng. pulp_ansible and pulp_container

because it depends on its plugins, so the 2nd run always break this idempotence test

Reply:
This task doesn't actually modify the system, so I am OK with it failing idempotency the 2nd time it is run. But we still want the task to run, and we still want to know if the system would be modified during `Install Pulp plugins via PyPI` as a result of this task generating a different requirements.in.